.github/workflows/gitleaks.yml

```diff
@@ -6,8 +6,8 @@ jobs:
   gitleaks_scan:
     runs-on: ubuntu-latest
     env:
-      REPO: https://github.com/MeasureAuthoringTool/madie-layout
-      REMOTE_EXCLUDES_URL: https://raw.githubusercontent.com/semanticbits/bmat-gitleaks-automation/master/madie-layout/gitleaks.toml
+      REPO: https://github.com/MeasureAuthoringTool/madie-patient
+      REMOTE_EXCLUDES_URL: https://raw.githubusercontent.com/semanticbits/bmat-gitleaks-automation/master/madie-patient/gitleaks.toml
```

Comment:
don't see `gitleaks.toml` file anywhere. is this something Ben adds or configured somewhere?

Reply:
I am trying to figure it out the same.

Reply:
I will have to create one, Would you like it to be cloned from a specific repo we had previously or would you like me to have a fresh config for this? @RohitKandimalla @adongare @gregory-akins

Reply:
I'm okay with the way we are doing it for other micro frontend

Reply:
Fixed

Reply:
Thanks @sb-benohe!